.github/workflows/release.yml

```diff
@@ -0,0 +1,88 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - feat/semantic-release # TODO - change the branch
+
+jobs:
+  bundler:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup | Clone codesandbox-client
+        uses: actions/checkout@v2
+        with:
+          repository: codesandbox/codesandbox-client
+
+      - uses: actions/cache@v2
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Setup | Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "10.23.2"
+          registry-url: "https://npm.pkg.github.com/"
+
+      - name: Setup | Install dependencies
+        run: yarn
+
+      - name: Prepare | Build dependencies
+        run: yarn build:deps
+
+      - name: Prepare | Build sandpack bundler
+        run: yarn build:sandpack
+
+      - name: Archive bundler artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: bundler
+          path: www/**/*.*
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: [bundler]
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+    steps:
+      - name: Setup | Checkout
+        uses: actions/checkout@v2
+        with:
+          # pulls all commits (needed for lerna / semantic release to correctly version)
+          fetch-depth: "0"
+
+      - name: Setup | Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12.22.4"
```

Comment:
are we locked with sandpack on node v12?

Reply:
Yes, we do. I think `yarn` requires at least 12, but I picked this values from here https://github.com/codesandbox/sandpack/blob/main/.nvmrc